art/preprocessing/standardisation_mean_std/standardisation_mean_std.py

```diff
@@ -66,18 +70,24 @@ def __call__(self, x: np.ndarray, y: Optional[np.ndarray] = None,) -> Tuple[np.n
                 "np.float32.".format(x.dtype)
             )
 
-        mean = np.asarray(self.mean, dtype=ART_NUMPY_DTYPE)
-        std = np.asarray(self.std, dtype=ART_NUMPY_DTYPE)
-
+        mean, std = broadcastable_mean_std(x, self.mean, self.std)
```

Comment:
Would it be possible to run `broadcastable_mean_std` outside of method forward? It should only be required once.

Reply:
See above.